tests/codegen/issues/issue-115385.rs

```diff
@@ -0,0 +1,50 @@
+// compile-flags: -O -Ccodegen-units=1
+// only-x86_64-unknown-linux-gnu
```

Comment:
Does this need to be limited to x64 linux? It doesn't look target specific.

Reply:
I'm not sure if this reproduces on other platforms, as that wasn't part of the original bug. I don't have other targets to verify whether this reproduces or not.

Reply:
We should avoid making tests target specific unless it's absolutely required, since it makes the development experience worse for devs on other platforms.

If you need to, you can make the test `no_core` and specify `--target x86_64-unknown-linux-gnu` instead, which works regardless of the host ([e.g.](https://github.com/rust-lang/rust/blob/2daacf5af965090b885287f1d40e13ff5db724cf/tests/codegen/aarch64-struct-align-128.rs#L4-L6)), but I don't think that will be necessary here--you can probably just delete this line.